### PR TITLE
Fix for self.hass.components.persistent_notification

### PR DESCRIPTION
**Pull request by bouwew**
## Summary by CodeRabbit

- **Refactor**
	- Simplified the creation of persistent notifications in the Plugwise integration for improved code readability without changing functionality.
- **Bug Fixes**
	- Resolved an issue with the integration of the Plugwise component accessing persistent notifications, ensuring correct display when the relevant attribute is present.
- **Documentation**
	- Updated the changelog to include a new section titled "Ongoing," reflecting ongoing improvements and fixes related to the Plugwise integration.

### CODE REVIEW

File: custom_components/plugwise/binary_sensor.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
     BinarySensorEntity,
     BinarySensorEntityDescription,
 )
+from homeassistant.components.persistent_notification import async_create
```
Comment (CoMPaTech) [inline custom_components/plugwise/binary_sensor.py:16]: Suggest adding `. # pw-beta: Notifications

Comment (CoMPaTech) [reply custom_components/plugwise/binary_sensor.py:16]: and maybe modify to:
`from homeassistant.components.persistent_notification import async_create as notification_async_create # pw-beta: Notifications` or something likewise?

Comment (bouwew) [reply custom_components/plugwise/binary_sensor.py:16]: I've also seen this: `import homeassistant.components.persistent_notification as pn`.
Then the line becomes: `pn.async_create()`.

Comment (bouwew) [reply custom_components/plugwise/binary_sensor.py:16]: Related to the comment, we already have a comment on line 170.

Comment (bouwew) [reply custom_components/plugwise/binary_sensor.py]: Yes OK I can add a comment for the new import. But nothing is needed further below.
Please confirm.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Versions from 0.40 and up
 
+## Ongoing
+
+- Fix for `integration 'plugwise' accesses hass.components.persistent_notification`.
+
```
[the remark this answers is not in the document]
Comment (bouwew) [reply CHANGELOG.md]: CHANGELOG-entry was improved.

**Review by CoMPaTech — APPROVED** [review]

**Pull request merged**